SchoolManagerSystem.Service/Validations/Implementations/RequestValidations.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SchoolManagerSystem.Service.Validations.Interfaces;
+using System.Text.RegularExpressions;
+
+namespace SchoolManagerSystem.Service.Validations.Implementations
+{
+	public class RequestValidations : IRequestValidations
```

Comment:
Create an helper class in Common and it should be a static class. 
you dont need interface for helpers.